packages/cosmic-proto/package.json

```diff
@@ -38,7 +38,7 @@
   "files": [
     "build",
     "LICENSE*",
-    "swingset/*"
+    "gen"
```

Comment:
This will work if we include both `swingset` (symlink sources) and `gen` (symlink targets).